pkg/config/config.go

```diff
@@ -583,6 +583,7 @@ func InitConfig(config Config) {
 	}
 
 	config.BindEnv("process_config.process_dd_url", "") //nolint:errcheck
+	config.BindEnvAndSetDefault("process_config.docker_host_ips", []string{})
```

Comment:
We should move the setting declaration to the `setupProcess` in the new `pkg/config/process.go` file and use `procBindEnv`. This helper function will automatically bind the env vars  `DD_PROCESS_CONFIG_DOCKER_HOST_IPS` and `DD_PROCESS_AGENT_DOCKER_HOST_IPS` to it.